spacy_llm/tests/pipeline/test_llm.py

```diff
@@ -62,8 +68,8 @@ def test_llm_serialize_disk():
         llm.from_disk(tmp_dir / "llm")
 
 
-@pytest.mark.skipif(has_openai_key is False, reason="OpenAI API key not available")
-def test_type_checking_valid() -> None:
+@pytest.mark.external
```

Comment:
In this case it's unfortunately still necessary, because the OpenAI REST backend calls into the API at init time to ensure that the authentication works (and the OpenAI REST backend is the default config we're testing here).

Reply:
Right. Thanks.